Command/HealthCheckCommand.php

```diff
@@ -58,6 +58,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $reporter = $this->getContainer()->get('liip_monitor.helper.raw_console_reporter');
         } else {
             $reporter = $this->getContainer()->get('liip_monitor.helper.console_reporter');
+            if (method_exists($reporter, 'suppressOKs')) {
```

Comment:
I had thought it may also be needed for the RawConsoleReporter or similar, but that always needs output. I'll delete that conditional.